doc/release-notes/release-notes-0.6.0.md

```diff
@@ -1,138 +1,49 @@
-bitcoin up-to-date.  Just type
-sudo apt-add-repository ppa:bitcoin/bitcoin
-in your terminal, then install the bitcoin-qt package.
-
-
-KNOWN ISSUES
-
-Shutting down while synchronizing with the network
-(downloading the blockchain) can take more than a minute,
-because database writes are queued to speed up download
-time.
-
-
-NEW FEATURES SINCE BITCOIN VERSION 0.5
-
-Initial network synchronization should be much faster
-(one or two hours on a typical machine instead of ten or more
-hours).
-
-Backup Wallet menu option.
-
-Bitcoin-Qt can display and save QR codes for sending
-and receiving addresses.
-
-New context menu on addresses to copy/edit/delete them.
-
-New Sign Message dialog that allows you to prove that you
-own a bitcoin address by creating a digital
-signature.
-
-New wallets created with this version will
-use 33-byte 'compressed' public keys instead of
-65-byte public keys, resulting in smaller
-transactions and less traffic on the bitcoin
-network. The shorter keys are already supported
-by the network but wallet.dat files containing
-short keys are not compatible with earlier
-versions of Bitcoin-Qt/bitcoind.
-
-New command-line argument -blocknotify=<command>
-that will spawn a shell process to run <command> 
-when a new block is accepted.
-
-New command-line argument -splash=0 to disable
-Bitcoin-Qt's initial splash screen
-
-validateaddress JSON-RPC api command output includes
-two new fields for addresses in the wallet:
-pubkey : hexadecimal public key
-iscompressed : true if pubkey is a short 33-byte key
-
-New JSON-RPC api commands for dumping/importing
-private keys from the wallet (dumprivkey, importprivkey).
-
-New JSON-RPC api command for getting information about
-blocks (getblock, getblockhash).
+https://github.com/chaintope/tapyrus-core/issues
 
-New JSON-RPC api command (getmininginfo) for getting
-extra information related to mining. The getinfo
-JSON-RPC command no longer includes mining-related
-information (generate/genproclimit/hashespersec).
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+https://github.com/chaintope/tapyrus-core/tarball/v0.6.0
 
 
+How to Upgrade
+==============
 
-NOTABLE CHANGES
+It is recommended to upgrade all older nodes to the latest release as it contains fixes to vulnerabilities. If you are running a node on tapyrus testnet follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-node-on-tapyrus-testnet) to sart a new Tapyrus v0.6.0 node.
 
-BIP30 implemented (security fix for an attack involving
-duplicate "coinbase transactions").
+If you are running a private tapyrus network using versions v0.3.0, v0.4.0 or v0.4.1, v0.5.0, v0.5.1, v0.5.2 shut it down. Wait until it has completely shut down. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-new-tapyrus-network) to sart a new Tapyrus v0.6.0 network. 
```

Comment:
```suggestion
If you are running a private tapyrus network using versions v0.3.0, v0.4.0 or v0.4.1, v0.5.0, v0.5.1, v0.5.2 shut it down. Wait until it has completely shut down. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-new-tapyrus-network) to start a new Tapyrus v0.6.0 network. 
```

Reply:
Also, I think it would be better to refer to it as "older versions" rather than listing all versions.